src/pages/index.js

```diff
@@ -146,22 +168,34 @@ export function updateProfile(profileObject) {
       name: newName,
       occupation: newOccupation,
     })
+    .then(() => {
+      userInfo.setUserInfo(newName, newOccupation);
+      editProfileFormValidator.disableSubmitButton();
```

Comment:
`.disableSubmitButton` is called when you open the modal. So, you don't need it here